book/scheduling.md

```diff
@@ -293,18 +294,28 @@ can't be accessed by another thread, so the lock does not need to be
 held while the task is running.
 
 ::: {.further}
-Event loops often map 1:1 to CPU threads within a single CPU process, but
-this is not required. For example, multiple event loops could be placed together
-on a single CPU thread with yet another scheduler on top of them that
-round-robins between them. It's useful to distinguish between: conceptual
-events, event queues and dependencies between them; and their implementation in
-a computer architecture. This way, the browser implementer (you!) has maximum
-ability to use more or less hardware parallelism as appropriate to the
-situation---some devices have more [CPU cores][cores] than others, or are more
-sensitive to battery power usage.
+Unfortunately, Python currently has a [global interpreter lock][gil],
```

Comment:
Isn't this go-further premature here, since we haven't yet talked about performance and parallelism?
Also, have we introduced the rendering pipeline at this point?

Reply:
Good point. I still think the GIL should go in front, when we introduce threads. Otherwise readers will feel a bit misled.

Reply:
I've edited the go-further to remove direct references to browser threads while still keeping the general point.